game-core/src/main/java/games/strategy/triplea/delegate/dataObjects/BattleRecord.java

```diff
@@ -13,6 +13,7 @@
  * The Purpose of this class is to record various information about combat,
  * in order to use it for conditions and other things later.
  */
+@SuppressWarnings("InconsistentCapitalization") // Field names cannot be changed to maintain backwards compatibility
```

Comment:
It would be preferable to apply the rule suppression to the fields themselves.  Unfortunately, Error Prone requires the suppression to be applied to the container of the identified fields.